install.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+set -euox pipefail
+
+# will return the current system uptime in milliseconds
+uptime_ms() {
+    # this is necessary to be able to return a value with milliseconds
+    # precision as busybox 'date' command does not support %N format
+    read up rest </proc/uptime
+    echo $(( 10 * (${up%.*}${up#*.}) ))
+}
+
+start_time=$(uptime_ms)
+
+finalizer() {
+    local dl="${download_dir:-}"
+    if [ -d "$dl" ]; then
+      rm -rf $dl
+    fi
+    code=$?
+    if [ $code -ne 0 ]; then
+        echo "ERROR: failed to install metrics extension: error code: $code"
+    fi
+    end_time=$(uptime_ms)
+    elapsed=$(( end_time-start_time ))
+    echo "Elapsed Time: $elapsed ms"
+    exit 0
+}
+trap finalizer EXIT
+
+# will download the extension respecting the max download
+# duration setting
+download_extension() {
+    mkdir -p $download_dir
+    echo "Downloading the UI extension..."
+    curl -Lf --max-time $download_max_sec $ext_url -o $ext_file
+    if [ "$checksum_url" != "" ]; then
+        echo "Validating the UI extension checksum..."
+        expected_sha=$(curl -Lf $checksum_url | grep "$ext_filename" | awk '{print $1;}')
+        current_sha=$(sha256sum $ext_file | awk '{print $1;}')
+        if [ "$expected_sha" != "$current_sha" ]; then
+            echo "ERROR: extension checksum mismatch"
+            exit 1
+        fi
+    fi
+    echo "UI extension downloaded successfully"
+}
+
+install_extension() {
+    echo "Installing the UI extension..."
+    cd $download_dir
+    local mime_type=$(file --mime-type "$ext_filename" | awk '{print $2}')
+    if [[ "$mime_type" == "application/gzip" ]]; then
+        tar -zxf $ext_filename
+    elif [[ "$mime_type" == "application/x-tar" ]]; then
+        tar -xf $ext_filename
+    else
+        echo "error: unsupported extension archive: $mime_type"
+        echo "supported formats: gzip and tar"
+        exit 1
+    fi
+    if [ ! -d "/tmp/extensions/resources" ]; then
+        mkdir -p /tmp/extensions/resources
+    fi
+    cp -Rf resources/* /tmp/extensions/resources/
+    echo "UI extension installed successfully"
+}
+
+
+## Script
+ext_enabled="${EXTENSION_ENABLED:-true}"
+if [ "$ext_enabled" != "true" ]; then
```

Comment:
```suggestion
if [[ "$ext_enabled" != "true" ]]; then
```

Reply:
I checked and verified that Busybox [actually ships](https://www.busybox.net/downloads/BusyBox.html) with `[[` tests. However I am not using any specific feature from `[[` so I am keeping it standardized with the rest of the script. More on that [here](https://superuser.com/questions/1533900/difference-between-and-or-and-in-bash)

Reply:
Thank for the clarification.